test_problems/mixGasTransport/mixGasTransport.cpp

```diff
@@ -129,17 +133,16 @@ int main(int argc, char** argv)
         grad_T[0] = (T2 - T1) / dist;
         grad_T[1] = (T3 - T1) / dist;
 
-        int log_level = 0;
-        Transport* tran = newTransportMgr("Mix", &g, log_level=0);
-        MixTransport* tranMix = dynamic_cast<MixTransport*>(tran);
-        g.setState_TPX(1500.0, pres, Xset.data());
+        auto tran = sol->transport();
+        auto tranMix = dynamic_pointer_cast<MixTransport>(tran);
```

Comment:
I don't think this is necessary -- what was here before was confused, to say the least -- there are no public methods of `MixTransport` that aren't just virtual methods available from a `Transport` pointer.